app/views/OverviewPageView.scala.html

```diff
@@ -0,0 +1,99 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+@import views.html.headerFooterTemplate.Layout
+
+@this(layout: Layout)
+
+@(isAgent: Boolean)(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+
+@breadcrumbs = {
+<div class="govuk-breadcrumbs ">
+ <ol class="govuk-breadcrumbs__list">
+  <li class="govuk-breadcrumbs__list-item">
+   <a class="govuk-breadcrumbs__link" href="#">Income Tax</a>
+  </li>
+  <li class="govuk-breadcrumbs__list-item">
+   <a class="govuk-breadcrumbs__link" href="#">Income Tax Submission</a>
+  </li>
+  <li class="govuk-breadcrumbs__list-item">
+   <a class="govuk-breadcrumbs__link">2020 to 2021 Income Tax</a>
+  </li>
+ </ol>
+</div>
+}
+
+@taskWithActionList(heading: String, action: String) = {
+<li>
+ <h2 class="app-task-list__section">
+  <span class="app-task-list__section-number">1. </span> @heading
+ </h2>
+ <ol class="app-task-list__items">
+  <li class="app-task-list__item">
+              <span class="app-task-list__task-name">
+                <a href="#">
+                  @action
+                </a>
+              </span>
+   <span class="hmrc-status-tag">Not Complete</span>
+  </li>
+ </ol>
+</li>
+}
+
+@taskWithCaption(heading: String, caption: String) = {
+ <li>
+  <h2 class="app-task-list__section">
+   <span class="app-task-list__section-number">2. </span> @heading
+  </h2>
+  <ul class="app-task-list__items">
+   <span class="govuk-caption-m">@caption</span>
+  </ul>
+ </li>
+}
+
+@overview = {
+<ol class="app-task-list">
+ @if(isAgent){
+ @taskWithActionList(messages("overview.agent.task1.heading"),messages("overview.individual.task1.income"))
+ @taskWithCaption(messages("overview.agent.task2.heading"),messages("overview.agent.task2.caption"))
+ @taskWithCaption(messages("overview.agent.task3.heading"),messages("overview.agent.task3.caption"))
+ @taskWithCaption(messages("overview.agent.task4.heading"),messages("overview.agent.task4.caption"))
+ }else{
+ @taskWithActionList(messages("overview.individual.task1.heading"),messages("overview.individual.task1.income"))
+ @taskWithCaption(messages("overview.individual.task2.heading"),messages("overview.individual.task2.caption"))
+ @taskWithCaption(messages("overview.individual.task3.heading"),messages("overview.individual.task3.caption"))
+ @taskWithCaption(messages("overview.individual.task4.heading"),messages("overview.individual.task4.caption"))
+ }
+</ol>
+}
+
+@layout(pageTitle = Some("income-tax-submission-frontend")) {
```

Comment:
The messages file can be used here for the page title (service.name) to replace the string val
<img width="178" alt="Screenshot 2020-09-25 at 15 20 11" src="https://user-images.githubusercontent.com/60218795/94278350-9c188b00-ff42-11ea-9070-3c739c30dd71.png">